client/components/pie-chart/index.js

```diff
@@ -0,0 +1,101 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { pie as d3Pie, arc as d3Arc } from 'd3-shape';
+import { assign, isEqual, sortBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import LegendItem from './legend-item';
+
+const NUM_COLOR_SECTIONS = 3;
+const SVG_SIZE = 300;
```

Comment:
These should probably be component props, with defaults values set

Reply:
The problem with doing that for `NUM_COLOR_SECTIONS` is there are css classes that make the coloring work, and if it exceeded 3 parts of the chart would be uncolored. I think that preventing the chart from being broken was a better approach.